src/engine/src/main.cpp

```diff
@@ -93,7 +94,7 @@ int main(int argc, char* argv[]) {
     }
 
   } catch (const InvalidArgumentException& e) {
-    CoincenterCmdLineOptions::PrintUsage(argv[0], e.what());
+    std::cerr << e.what() << std::endl;
```

Comment:
In this case, standard scripts do not print the whole `--help` but only the error itself, so I decided to keep same behavior. This way, the error is clearer and we do not need to scroll up to see the message